components/governance/org.wso2.carbon.governance.taxonomy/src/main/java/org/wso2/carbon/governance/taxonomy/internal/TaxonomyServiceComponent.java

```diff
@@ -43,15 +63,63 @@
      *
      * @param context OSGi component context.
      */
-    protected void activate(ComponentContext context) {
+    protected void activate(ComponentContext context)
+            throws UserStoreException, RegistryException, ParserConfigurationException, SAXException,
+            XPathExpressionException, IOException {
```

Comment:
Do we need to throw exceptions in activate method. we need to handle inside activate method